cpp/ql/src/semmle/code/cpp/security/FunctionWithWrappers.qll

```diff
@@ -3,9 +3,14 @@ import PrintfLike
 private import TaintTracking
 
 private
+bindingset[index]
```

Comment:
Using `bindingset` makes the function inline, which can make it harder to predict how it'll be optimised in future contexts. Do these unnamed parameters at least have a `Parameter` object? Then you should be able to add `exists(func.getParameter(index))` after the `or` and remove the `bindingset` annotation.

Reply:
I tried that and I didn't get the same results (and the test fails).  I believe `getParameter` had no results.

Because this predicate is `private`, any non-local performance effects due to inlining should be localized to within this library.

Reply:
👍